routes/feed.js

```diff
@@ -0,0 +1,37 @@
+const Feed = require('feed')
+const striptags = require('striptags')
+
+module.exports = function feedHandler (req, res, next) {
+  let feed = new Feed({
+    title: 'Electron',
+    description: 'Build cross platform desktop apps with JavaScript, HTML, and CSS.',
+    id: 'http://electronjs.org/',
+    link: 'http://electronjs.org/',
+    generator: 'Electron website',
+    feedLinks: {
+      json: 'https://electronjs.org/feed.json',
+      atom: 'https://electronjs.org/feed.xml',
+    }
+  })
+  req.context.posts.forEach(function (post) {
+    feed.addItem({
+      id: `https://electronjs.org${post.href}`,
+      title: post.title,
+      content: post.content,
+      description: `${striptags(post.content).split(' ').slice(0, 200).join(' ')} [...]`,
```

Comment:
Maybe the `[...]` should only be appended to the string if there is actually truncated content?